sass/components/_chart-title.scss

```diff
@@ -1,122 +1,123 @@
 .cq-chart-title {
+    display: inline-block;
     top: 1em;
     z-index: 41;
     margin: 0.2em 0.8em;
 
-    &.stxMenuActive {
-        box-shadow: none !important;
-
-        .cq-menu-btn {
-            .cq-symbol-select-btn {
-                @include themify($themes) {
-                    border: 1px solid themed('menuActiveBorder');
-                }
-
-                .cq-symbol-dropdown {
-                    transform: rotate(180deg) !important;
-                }
-            }
-        }
-    }
     .cq-menu-btn {
-        padding: 0 !important;
-        margin: 0.2em 0em;
-
-        .cq-symbol-select-btn {
-            font-family: $base-font;
-            padding: 0.3em 0.5em 0.3em 1em;
+        > .cq-symbol-select-btn {
             cursor: pointer;
-            position: relative;
-            display: flex;
-            align-items: center;
 
             &:hover {
                 @include themify($themes) {
                     border: solid 1px themed('menuActiveBorder');
                 }
             }
-            @include border-radius();
+            > .ic-icon.cq-symbol-dropdown {
+                display: block;
+                margin-right: 0px;
+                width: 24px;
+                transform: scale(1);
+            }
+        }
+    }
+    &.stxMenuActive > .cq-menu-btn {
+        > .cq-symbol-select-btn {
             @include themify($themes) {
-                color: themed('ChartTitleText');
-                background: themed('ChartTitleBg');
-                border: solid 1px themed('ChartTitleBorder');
+                border: 1px solid themed('menuActiveBorder');
             }
 
-            > .ic-icon, > .ic-frx, > .ic-wld, > .ic-otc {
-                height: $large-icon-size;
-                width: $large-icon-size;
-                margin-right: 12px;
-                transform: scale(1.3);
-
-                > svg {
-                    height: $large-icon-size;
-                }
+            > .ic-icon.cq-symbol-dropdown {
+                transform: rotate(180deg);
```

Comment:
In general I'm opposed to using `!important` as much as possible in the code base. Either make the query more specific or restructure the classes to give it precedence.